### PR TITLE
Request Form Cosmetic Updates

### PR DESCRIPTION
**Pull request by vishk1**
Issue #381 
- added new text fields 
- created new "sections" divided by each text field
- fixed spacing in-between sections, form & text
- fixed overflow/scroll error by adding overflow-auto
- commented out old text msg block

### CODE REVIEW

**Comment by vishk1** [comment]
image

**Comment by vishk1** [comment]
- added "title" input
- added "primary language" button as a dropdown with 'down' icon

**Comment by vishk1** [comment]
image

File: src/pages/home/RequestAccess.vue [hunk 1]
```diff
@@ -17,7 +17,12 @@
                 })
               }}
             </div>
-            <div>{{ $t('requestAccess.enter_existing_user_email_msg') }}</div>
+            <base-text variant="body" weight="700" class="mt-4 mb-1">
+              ~~To join an organization, you should be invited by an existing
+              member. You can use this form to <br />
+              request access from an existing user by entering their email here.
```
Comment (BradenM) [inline src/pages/home/RequestAccess.vue:23]: these need to be wrapped in calls to `$t` to trigger the translations

**Comment by vishk1** [comment]
- added translation calls around text

**Pull request merged**